sno/merge.py

```diff
@@ -156,18 +157,23 @@ def complete_merging_state(ctx):
     moves the repo from merging state back into the normal state, with the branch
     HEAD now at the merge commit. Only works if all conflicts have been resolved.
     """
-    repo = ctx.obj.repo
-    if not is_ongoing_merge(repo):
-        raise InvalidOperation('Repository is not in "merging" state.')
+    repo = ctx.obj.get_repo(
+        allowed_states=[RepoState.MERGING],
+        bad_state_message='`sno merge --continue` only works when sno repo is in "merging" state.',
+    )
     merge_index = MergeIndex.read_from_repo(repo)
     if merge_index.unresolved_conflicts:
         raise InvalidOperation(
```

Comment:
feels like these errors for invalid state transitions should probably have tests. at least for the exit codes, if not the actual messages

Reply:
Done